modules/investigating-etcd-installation-issues.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-installations.adoc
+
+[id="investigating-etcd-installation-issues_{context}"]
+= Investigating etcd installation issues
+
+If you experience etcd issues during installation, you can check etcd Pod status and collect etcd Pod logs. You can also verify etcd DNS records and check DNS availability on master nodes.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+* You have installed the OpenShift CLI (`oc`).
+* The `ssh-agent` process is active on your computer, and you provided both the `ssh-agent` process and the installation program with the same SSH key.
+* If you tried to install a cluster on infrastructure that you provisioned, you must have the fully qualified domain names of the bootstrap node.
+
+.Procedure
+
+. Check the status of etcd Pods.
+.. Review the status of Pods in the `openshift-etcd` namespace:
++
+----
+$ oc get pods -n openshift-etcd
+----
++
+.. Review the status of Pods in the `openshift-etcd-operator` namespace:
++
+----
+oc get pods -n openshift-etcd-operator
+----
+
+. If any of the Pods listed by the previous commands are not showing a `Running` or a `Completed` status, gather diagnostic information for the Pod.
+.. Review events for the Pod:
++
+----
+$ oc describe pod/<pod_name> -n <namespace>
+----
++
+.. Inspect the Pod's logs:
++
+----
+$ oc logs pod/<pod_name> -n <namespace>
+----
++
+.. If the Pod has more than one container, the preceding command will create an error, and the container names will be provided in the error message. Inspect logs for each container:
++
+----
+$ oc logs pod/<pod_name> -c <container_name> -n <namespace>
+----
+
+. If the API is not functional, review etcd Pod and container logs on each master node by using SSH instead. Replace `<master-node>.<cluster_name>.<sub_domain>.<domain>` with appropriate values.
+.. List etcd Pods on each master node:
++
+----
+$ ssh core@<master-node>.<cluster_name>.<sub_domain>.<domain> sudo crictl pods --name=etcd-member
+----
++
+.. For any Pods not showing `Ready` status, inspect Pod status in detail. Replace `<pod_id>` with the Pod's ID listed in the output of the preceding command:
++
+----
+$ ssh core@<master-node>.<cluster_name>.<sub_domain>.<domain> sudo crictl inspectp <pod_id>
+----
++
+.. List containers related to a Pod:
++
+----
+$ ssh core@<master-node>.<cluster_name>.<sub_domain>.<domain> sudo crictl ps --pod=<pod_id>
```

Comment:
I tried to run this command, could not list filtering by pod_id, but actually this pod has several containers.

Reply:
It looks like some of my comments in this PR have been lost for some reason. To reiterate, I filed https://bugzilla.redhat.com/show_bug.cgi?id=1858239 in relation to the `--pod` issue you described above. I have commented out the command in the document and added a TODO to include it again once the BZ is resolved. I have in the meantime added a grep alternative in the doc.